tests/db/test_mysql.py

```diff
@@ -38,6 +38,25 @@ class MockMySQLConnection(MySQLConnectionAbstract):
     is_connected = MagicMock()
     ping = MagicMock()
     rollback = MagicMock()
+    cmd_change_user = MagicMock()
+    cmd_debug = MagicMock()
+    cmd_init_db = MagicMock()
+    cmd_ping = MagicMock()
+    cmd_process_kill = MagicMock()
+    cmd_query = MagicMock()
+    cmd_query_iter = MagicMock()
+    cmd_quit = MagicMock()
+    cmd_refresh = MagicMock()
+    cmd_reset_connection = MagicMock()
+    cmd_shutdown = MagicMock()
+    cmd_statistics = MagicMock()
+    cmd_stmt_close = MagicMock()
+    cmd_stmt_execute = MagicMock()
+    cmd_stmt_prepare = MagicMock()
+    cmd_stmt_reset = MagicMock()
+    cmd_stmt_send_long_data = MagicMock()
+    connection_id = MagicMock()
+    get_row = MagicMock()
```

Comment:
Could you sort these by name like the original list of mocks?  When it comes to things like these, not having them sorted means it's much harder to work on and maintain these in future.

Reply:
yeah, agreed. I'll sort it.

Reply:
Resolving since this change was made in the latest commit.